Velaptor/InternalExtensionMethods.cs

```diff
@@ -177,6 +177,17 @@ public static string TrimDirSeparatorFromEnd(this string value)
             return value;
         }
 
+        /// <summary>
+        /// Converts the given <paramref name="path"/> to a cross platform path.
+        /// </summary>
+        /// <param name="path">The file or directory path.</param>
+        /// <returns>The cross platform version of the <paramref name="path"/>.</returns>
+        /// <returns>
+        ///     This just takes all '\' characters and changes them to '/' characters.
+        ///     The '/' directory separator is valid on windows and linux systems.
```

Comment:
Refactor total code base to capitalize Windows and Linux
```suggestion
        ///     The '/' directory separator is valid on Windows and Linux systems.
```